sqlpad.yaml

```diff
@@ -0,0 +1,47 @@
+package:
+  name: sqlpad
+  version: 7.1.3
+  epoch: 0
+  description: Web-based SQL editor. Legacy project in maintenance mode.
+  copyright:
+    - license: MIT
+  dependencies:
+    runtime:
+      - nodejs-18
+      - yarn
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - build-base
+      - yarn
+      - nodejs-18
```

Comment:
just checking - does this have to be pinned to `nodejs-18`?  Am wondering if `nodejs` would be fine which today is nodejs-20.  Just means we don't have to manage this in the future.

Reply:
As much I remember, when I tried last, it failed and had to check sqlpad and they supported version 18 that's why I pinned it, let me re-try

Reply:
Here https://github.com/sqlpad/sqlpad/blob/master/.nvmrc

Reply:
No worries at all, just done, thanks